### PR TITLE
[OSDOCS-5101]: Replacement control plane nodes in correct group

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.13

Issue:
[OCPCLOUD-1562](https://issues.redhat.com//browse/OCPCLOUD-1562) | [BZ#1970464](https://bugzilla.redhat.com/show_bug.cgi?id=1970464) | [OSDOCS-5101](https://issues.redhat.com//browse/OSDOCS-5101)

Link to docs preview:
[Bugfix writeup](https://56525--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-bug-fixes)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jeana-redhat** [comment]
@RadekManak PTAL :pray:

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -408,6 +408,9 @@ In {product-title} 4.12, support for Kuryr on clusters that run on {rh-openstack
 [id="ocp-4-13-cloud-compute-bug-fixes"]
 ==== Cloud Compute
 
+* For some configurations of Google Cloud Platform clusters, the internal load balancer uses instance groups that are created by the installation program. Previously, when a control plane machine was replaced manually, the new control plane node was assigned to a compute instance group rather than a control plane instance group. To resolve the issue, administrators had to manually move the control plane machine to the correct instance group by using the Google Cloud console. With this release, replacement control plane nodes are assigned to the correct instance group.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1970464[*BZ#1970464*], link:https://issues.redhat.com/browse/OCPCLOUD-1562[*OCPCLOUD-1562*])
+
```
Comment (RadekManak) [inline release_notes/ocp-4-13-release-notes.adoc:413]:
```suggestion
* For some configurations of Google Cloud Platform clusters, the internal load balancer uses instance groups that are created by the installation program. Previously, when a control plane machine was replaced manually, the new control plane node was not assigned to a compute instance group preventing the node being reachable via the internal load balancer. To resolve the issue, administrators had to manually assign the control plane machine to the correct instance group by using the Google Cloud console. With this release, replacement control plane nodes are assigned to the correct instance group.
(link:https://bugzilla.redhat.com/show_bug.cgi?id=1970464[*BZ#1970464*], link:https://issues.redhat.com/browse/OCPCLOUD-1562[*OCPCLOUD-1562*])

```

Comment (jeana-redhat) [reply release_notes/ocp-4-13-release-notes.adoc]: Thanks for the additional detail on this - let me know what you think of the revision :)

**Comment by jeana-redhat** [comment]
@miyadav PTAL

**Comment by miyadav** [comment]
Thanks @jeana-redhat , LGTM

**Comment by bscott-rh** [comment]
post-mortem LGTM.

**Pull request merged**